app/react/Relationships/utils/specs/relationshipsUtils.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { filterVisibleConnections } from 'app/Relationships/utils/relationshipsUtils';
+
+describe('relationshipsUtils', () => {
+  describe('filterVisibleConnections', () => {
+    it('should filter connectionGroups based in hub availability of entities', () => {
```

Comment:
this test is really dificult to understand. i need to know more in order to understand what is filtered and why.

Reply:
I rewrite the test in order to make it more readable.